### PR TITLE
build/configs/rtl8730e: Add PSRAM config

### PR DESCRIPTION
**Pull request by ziyik**
-Add PSRAM config for flat app development use
-Maintain DDR config

### CODE REVIEW

**Review by sunghan-chang — COMMENTED** [review]
We have no plan to use the flat build for rtl8730e. Do you still need those defconfigs?

**Comment by ziyik** [comment]
> We have no plan to use the flat build for rtl8730e. Do you still need those defconfigs?

Yes, it's for our internal development use.

**Comment by sunghan-chang** [comment]
@ziyik Build break because of ci configuration.
Let's modify that with new commit in this PR.

And `build/configs/rtl8730e: Add PSRAM config` is difficult to know content.
How about `build/configs/rtl8730e: add flat_dev_psram`? and could you split a commit for renaming flat_dev to flat_dev_ddr?

**Comment by ziyik** [comment]
> @ziyik Build break because of ci configuration. Let's modify that with new commit in this PR.
> 
> And `build/configs/rtl8730e: Add PSRAM config` is difficult to know content. How about `build/configs/rtl8730e: add flat_dev_psram`? and could you split a commit for renaming flat_dev to flat_dev_ddr?

All done

**Comment by sunghan-chang** [comment]
> @ziyik Build break because of ci configuration.
> Let's modify that with new commit in this PR.

As I said above, let's do changing ci configuration with new commit in this PR. Let's split defconfig change and ci.yml change.

**Comment by ziyik** [comment]
> > @ziyik Build break because of ci configuration.
> > Let's modify that with new commit in this PR.
> 
> As I said above, let's do changing ci configuration with new commit in this PR. Let's split defconfig change and ci.yml change.

The second commit changes the defconfig name and ci.yml.
Do you mean split these into 2 commits?

**Comment by sunghan-chang** [comment]
> > > @ziyik Build break because of ci configuration.
> > > Let's modify that with new commit in this PR.
> > 
> > 
> > As I said above, let's do changing ci configuration with new commit in this PR. Let's split defconfig change and ci.yml change.
> 
> The second commit changes the defconfig name and ci.yml. Do you mean split these into 2 commits?

Right. So totally three commits.
1. rename
2. ci change
3. add new defconfig

**Comment by ziyik** [comment]
> > > > @ziyik Build break because of ci configuration.
> > > > Let's modify that with new commit in this PR.
> > > 
> > > 
> > > As I said above, let's do changing ci configuration with new commit in this PR. Let's split defconfig change and ci.yml change.
> > 
> > 
> > The second commit changes the defconfig name and ci.yml. Do you mean split these into 2 commits?
> 
> Right. So totally three commits.
> 
> 1. rename
> 2. ci change
> 3. add new defconfig

So the comment needed to follow the above order?
First, rename config, second ci change then add new defconfig?
ci change should be the last commit, so it can trigger correctly, is that correct?

**Comment by sunghan-chang** [comment]
anything is ok.

File: .circleci/config.yml [hunk 1]
```diff
@@ -145,7 +145,17 @@ jobs:
...
+    machine:
+      image: default
+    working_directory: ~/TizenRT
+    steps:
+      - attach_workspace:
+          at: ~/TizenRT
+      - *docker-cp
+      - *build-job
+
+  rtl8730e/flat_dev_psram:
```
Comment (sunghan-chang) [inline .circleci/config.yml:158]: As you said, they are Realtek internal dev purposes. Even we run just one loadable configuration due to ci runtime, it's difficult to run all. Please keep 1 configuration for them.

**Review by sunghan-chang — APPROVED** [review]

**Pull request merged**